Fix E2E tests & mssql init & housekeeping .env files in git

weilu opened the pull request.

weilu committed:
fix: femove duplicate init & ports

weilu committed:
git ignores openSearch and redis .env files

delcroip commented on compose.mssql.yml:
```diff
@@ -17,7 +17,6 @@ services:
       timeout: 5s
       retries: 5
       start_period: 100s
-    init: true
```
I don't get the link with indexes

weilu replied:
PR title was a typo, should have said init. Fixed.

weilu committed:
Update ci to use the latest set of env files

weilu committed:
Check backend up and running before running cypress test

Also test on develop tag for now,
until coreMIS FE is updated with dhparam.pem commit from develop

weilu committed:
Also gitignore .env.database

weilu committed:
Add default CSRF_TRUSTED_ORIGINS to .env.example

As it is required for the docker deployment to work

delcroip commented on .env.example:
```diff
@@ -52,3 +52,4 @@ DJANGO_LOG_HANDLER=debug-log
 # should the database be migrated at every container startup. Will be done anyway if $SITE_ROOT=api
 DJANGO_MIGRATE=True
 
+CSRF_TRUSTED_ORIGINS=http://localhost
```
```suggestion
DOMAIN=localhost
```
